graph/pkg/service/v0/drives.go

```diff
@@ -191,15 +191,13 @@ func (g Graph) CreateDrive(w http.ResponseWriter, r *http.Request) {
 		Quota: getQuota(drive.Quota, g.config.Spaces.DefaultQuota),
 	}
 
+	opaque := make(map[string]*types.OpaqueEntry, 1)
 	if drive.Description != nil {
-		csr.Opaque = &types.Opaque{
-			Map: map[string]*types.OpaqueEntry{
-				"description": {
-					Decoder: "plain",
-					Value:   []byte(*drive.Description),
-				},
-			},
+		opaque["description"] = &types.OpaqueEntry{
+			Decoder: "plain",
+			Value:   []byte(*drive.Description),
 		}
+		csr.Opaque = &types.Opaque{Map: opaque}
```

Comment:
I think we sohuld use the reva `pkg/utils` package to append a plain opaque property: https://github.com/cs3org/reva/blob/d217886c962b05ab7aa751923016e3ccb4dd7534/pkg/utils/utils.go#L374-L387

Reply:
ok, will do. I don't like that we cannot define the length of the map. So the map needs to reallocate everytime when we add a property-

Reply:
It's fine if it's only a few items.